python/pdstools/adm/ADMDatamart.py

```diff
@@ -1366,26 +1368,36 @@ class into the Health Check. Simply set the `predictorCategorization` to a
         """
 
         def delete_temp_files(working_dir, files):
-            for f in ["params.yaml", "HealthCheck.qmd", "HealthCheck.ipynb", "log.txt"]:
+            for f in ["params.yaml", "HealthCheck.qmd", "HealthCheck.ipynb",
+                      "HealthCheckModel.qmd", "HealthCheckModel.ipynb","log.txt"]:
                 try:
                     os.remove(f"{working_dir}/{f}")
                 except:
                     pass
             for f in files:
-                os.remove(f)
+                try:
+                    os.remove(f)
+                except:
+                    pass
+
+        working_dir, output_location = Path(working_dir), Path(output_location)
 
         from pdstools import __reports__
 
         verbose = kwargs.get("verbose", self.verbose)
 
-        if self.modelData is None or self.predictorData is None:
-            raise AssertionError("Needs both model and predictor data.")
+        if self.modelData is None:
+            raise AssertionError("Needs model data.")
+        if self.predictorData is None:
+            quarto = "HealthCheckModel.qmd"
```

Comment:
Please call this variable something like 'healthcheck_file' or something, not quarto. Quarto is a library, so let's not conflict